src/test/java/redis/clients/jedis/tests/commands/BitCommandsTest.java

```diff
@@ -194,12 +194,10 @@ public void testBitfield() {
 
   @Test
   public void testBinaryBitfield() {
-    List<byte[]> responses = jedis.bitfield(SafeEncoder.encode("mykey"), SafeEncoder.encode("INCRBY"),
-        SafeEncoder.encode("i5"), SafeEncoder.encode("100"), SafeEncoder.encode("1"),
-        SafeEncoder.encode("GET"), SafeEncoder.encode("u4"), SafeEncoder.encode("0")
-    );
-    assertEquals(1L, responses.get(0));
-    assertEquals(0L, responses.get(1));
+    List<Long> responses = jedis.bitfield("mykey".getBytes(), "INCRBY".getBytes(), "i5".getBytes(),
```

Comment:
I'd like to have a consistent way to convert string to binary, and if my memory is right, someone complained that this way doesn't work on several charsets.